### PR TITLE
Add Xoriant as new partner

### PR DESCRIPTION
**Pull request by Ram-Kumaar-ML**
### Description
[Describe what this change achieves]
 
### Issues Resolved
[List any issues this PR will resolve]

### Check List
- [ ] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

**Comment by Ram-Kumaar-ML** [comment]
@krisfreedain - Created a new PR and figured out the DCO issue . For some reason, I'm unable to do a local build of the site and have updated the xoriant.md file with the error fixed (hopefully). Please kindly check.

**Comment by krisfreedain** [comment]
thanks @Ram-Kumaar-ML DCO works!

I'm getting an odd error
"project-website/_partners/xoriant.md: (<unknown>): found character that cannot start any token while scanning for the next token at line 47 column 1" 
so I'll investigate that and see what we need to fix.

File: _partners/xoriant.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+# Comma delimited list of one or more industries that the partner specializes in serving.
+industries:  Manufacturing and Automotive, Supply Chain, Consumer Services, Education, Energy and Utilities, Financial Services, Healthcare, Media and Entertainment, Public sector, Retail and e-commerce, Software and Technology
+
+# If the partner only has one office location then use the main_office_location key
+# otherwise use the multiple_office_locations key.
+# In the template rendering multiple_office_locations takes precedent,
+# and if both are used only the multiple_office_locations data will be rendered
+# in the template omitting the main_office_location data.
+main_office_location:
```
Comment (astephanus) [inline _partners/xoriant.md:46]: @Ram-Kumaar-ML The format for the address requires a multiline string format with new lines preserved using the `|` character. Here's a SO discussion that gives some straight forward detail about the variants. https://stackoverflow.com/questions/3790454/how-do-i-break-a-string-in-yaml-over-multiple-lines
Also, refer to `_partners/_sample.md`, or real world examples in many other `*.md` within the `_partners` collection.

**Review by astephanus — CHANGES_REQUESTED** [review]
Fixing the addrss format will resolve the error.

**Comment by pajuric** [comment]
@Ram-Kumaar-ML - checking in on this and seeing if there is anything I can do to help get your company updated.

File: _partners/xoriant.md (hunk 1 above)
Comment (krisfreedain) [reply _partners/xoriant.md:46]: @astephanus - I'm still seeing that odd error when I pull it down locally - can you give it another look? 

"_partners/xoriant.md: (<unknown>): found character that cannot start any token while scanning for the next token at line 47 column 1"

**Comment by pajuric** [comment]
@Ram-Kumaar-ML - Just checking in to see if you're able to help resolve this. Looks like there's still an error.

**Review by nateynateynate — APPROVED** [review]
Made an executive decision and fixed the frontmatter for Xoriant to take out the tabs. Not sure why Jekyll hates this.

**Pull request merged**